### PR TITLE
[prathamesh_m009] Added /info endpoint

### PR DESCRIPTION
**Pull request by prathamesh-mutkure**
This commit is part of this [GCI Task](https://codein.withgoogle.com/dashboard/task-instances/6521601542062080/)

Thank you!

### CODE REVIEW

**Review by Akayeshmantha — COMMENTED** [review]
Good work

File: index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+var data  = require('./package.json');
+
+// create a route for the app
+app.get('/', (req, res) => {
+  res.send('Hello World');
+});
+
+// create a route for the /info endpoint
+app.get('/info', (req, res) => {
+    res.send("{ serverName: '" + data.name + "', serverVersion: '" + data.version + "' }");
```
Comment (suthagar23) [inline index.js]: Can't we create a JSON, and then send it or stringify and send it? Did you try?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+});
+
+// make the server listen to requests
+app.listen(PORT, () => {
+  console.log(`Server running at: http://localhost:${PORT}/`);
+});
```
Comment (suthagar23) [inline index.js]: Need a new line at the end of the file

File: index.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const express = require('express');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 3000;
+
+// JSON Data
+var data  = require('./package.json');
```
Comment (suthagar23) [inline index.js]: better to rename this variable as `packageData`, Because it should be specific among the file.

**Review by suthagar23 — CHANGES_REQUESTED** [review]

File: index.js (hunk 3 above)
Comment (prathamesh-mutkure) [reply index.js]: Thanks, Done!

File: index.js (hunk 2 above)
Comment (prathamesh-mutkure) [reply index.js]: Thanks for pointing that out, just researched about importance of new line at end of file in unix :)

File: index.js (hunk 1 above)
Comment (prathamesh-mutkure) [reply index.js]: Well, yes the task asks to return JSON, but stringifying it causes double quotes to come into play, will it be fine against single quotes required for tasks?

File: index.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+// create a route for the app
+app.get('/', (req, res) => {
+  res.send('Hello World');
+});
+
+// create a route for the /info endpoint
+app.get('/info', (req, res) => {
+
+  var jsonOutputText = '{ "serverName": "' + packageData.name + '", "serverVersion": "' + packageData.version + '" }';
```
Comment (suthagar23) [inline index.js]: What do you think about, 
```
{ serverName: ..., 
  serverVersion: ...
}
```
Then you don't need to parse it also

**Review by suthagar23 — CHANGES_REQUESTED** [review]

File: index.js (hunk 4 above)
Comment (prathamesh-mutkure) [reply index.js]: If you mean directly returning the string without parsing, Done!

File: index.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  res.send(jsonOutputText);
```
Comment (suthagar23) [inline index.js]: What do you think about,
```
res.send( { serverName: packageData.name, serverVersion: packageData.version } )
```

File: index.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+});
+
+// route for displaying all products
+app.get('/products/all', (req, res) => {
```
Comment (suthagar23) [inline index.js]: Not related to this task

**Review by suthagar23 — CHANGES_REQUESTED** [review]

File: index.js (hunk 6 above)
Comment (prathamesh-mutkure) [reply index.js]: I'm sorry, I pushed from wrong branch (which was for 3rd react task)

File: index.js (hunk 5 above)
Comment (prathamesh-mutkure) [reply index.js]: Thanks for suggestation, Done!

**PS**: The output is identical

**Pull request merged**